lepton/gcp.go

```diff
@@ -139,6 +149,12 @@ func (p *GCloud) CreateInstance(ctx *Context) error {
 	return nil
 }
 
+func createArchiveExternal(archive string, files []string) error {
+	cmd := exec.Command("tar", "-czvf", archive, filepath.Base(files[0]))
```

Comment:
as mentioned above - need to figure out what's going on w/native go tar and should prob. use that instead